.vscode/cspell.json

```diff
@@ -1040,6 +1043,13 @@
         "vertica",
         "xero",
         "Asana",
+        "Ddls",
+        "Ename",
+        "Evalue",
+        "Informations",
```

Comment:
Did you mean to add just 
- Informations
- Ename
- Evalue

Which schemas/contracts in swagger are these referring to?

Reply:
@piyushdubey we will check the words after generating SDK. These words are not correct English so the check will complain. I add these words to cspell.json to let the check ignore them.

What Anne means is we might review these names to make them correct instead of just ignoring the check.

Reply:
Deleted in this release.